src/main/java/ai/elimu/rest/v2/crowdsource/ContributorsRestControllerWeb3.java

```diff
@@ -69,7 +69,7 @@ public String handleGetRequest(
             logger.info("jsonResponse: " + jsonResponse);
             return jsonResponse;
         }
-        
+        providerIdWeb3 = providerIdWeb3.toLowerCase();
```

Comment:
Keep the empty line between the two code blocks:
```suggestion
        providerIdWeb3 = providerIdWeb3.toLowerCase();
        
```